Test/Framework/Graphics/GraphicsDeviceTest.cs

```diff
@@ -9,6 +9,7 @@
 using Microsoft.Xna.Framework.Graphics;
 using MonoGame.Tests.ContentPipeline;
 using NUnit.Framework;
+using OpenGL;
```

Comment:
I wonder why this isn't breaking the Windows DirectX build.  We wouldn't expect this namespace to exist outside of `DESKTOPGL` right?

Reply:
IWindowInfo and IGraphicsContext are in the OpenGL namespace and they're included (but not used) in MG DX. If you want I'll exclude them from MG DX in this PR.

Reply:
I see... well for now we don't need to change this PR then.

Ideally we open a new PR and first rename that namespace from `OpenGL` to `MonoGame.OpenGL` to make it clear it is a MG thing and not some other asssembly.  Then it would also be good for those to be only added to the appropriate OGL platforms.

Reply:
Also noticed this...

![image](https://user-images.githubusercontent.com/103522/30169498-321ba178-93b2-11e7-9750-5c2df4896e56.png)

Probably none of that should be part of the public API... but it is as it shows up in our documentation now.

That should likely all be cleaned up and made private/internal.

Reply:
@Jjagg - @KonajuGames handled this...  https://github.com/MonoGame/MonoGame/pull/5927